src/CommonItilObject_Item.php

```diff
@@ -1576,4 +1577,36 @@ public static function dropdownAllDevices(
         }
         return $rand;
     }
+
+    /**
+     * ITIL tabs for generic assets should only be displayed if the asset already
+     * has associated ITIL items OR if the current user profile is allowed to
+     * link this asset to ITIL items
+     *
+     * @param Asset $asset
+     *
+     * @return bool
+     */
+    protected function shouldDisplayTabForAsset(Asset $asset): bool
+    {
+        // Always display tab if the current profile is allowed to link ITIL
+        // items to this asset
+        if (
+            in_array(
+                $asset::class,
+                $_SESSION["glpiactiveprofile"]["helpdesk_item_type"] ?? []
+            )
+        ) {
+            return true;
+        }
+
+        // Only show if at least one item is already linked
+        return countElementsInTable(
+            static::getTable(),
+            [
+                'itemtype' => $asset::getType(),
+                'items_id' => $asset->getId(),
+            ]
+        ) > 0;
```

Comment:
If the tab is not shown when there is no linked items, it will not anymore possible to use the `New Ticket for this item` feature:
![image](https://github.com/glpi-project/glpi/assets/33253653/2124c3b7-6b83-4a39-9aa1-beba0cb7f9ee)

Reply:
The reason for this is that this is a basic capacity that can't be disabled.
If the tab is always visible, users might be stuck with 3 useless extra tabs on their assets.

I thought only showing the tab if the item is associable to a ticket or if there are already linked item was a good compromise to work around this.

> If the tab is not shown when there is no linked items, it will not anymore possible to use the New ticket for this item feature:

This feature is only enable if the asset is associable to a ticket, which in this case the tab will always be shown so no issues here.

Reply:
> users might be stuck with 3 useless extra tabs on their assets. It kinda defeat the whole "customizable" part.

Just like it's always been... It's not a recent subject.

My point is not this is a good or a bad idea; but if you want to propose a solution for "too many tabs", it's not something to add in this kind of PR; it has to be validated as a separate feature.

Reply:
Also, having a distinct behaviour for generic asset and other asset may confuse end users. Indeed, they may think that lack of tab presence is a bug rather than a missing profile configuration.
If we want to change this behaviour, it would be preferable to do it globally.